app/src/test/kotlin/com/exallium/todoapp/editnote/EditNotePresenterTest.kt

```diff
@@ -73,7 +70,7 @@ class EditNotePresenterTest {
         // THEN
         verify(screenBundleHelper).getNoteId(bundle)
     }
-
+    
```

Comment:
github complained http://stackoverflow.com/questions/5813311/no-newline-at-end-of-file